hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -194,6 +194,18 @@ private URL changePrefixFromBlobtoDfs(URL url) throws InvalidUriException {
     return url;
   }
 
+  private URL changePrefixFromDfsToBlob(URL url) throws InvalidUriException {
+    if (url.toString().contains(WASB_DNS_PREFIX)) {
```

Comment:
Wont this need a first check of enableBlobEndpoint is true ?

Reply:
1. This method will be called only by those methods which come in the flow of blob-endpoint operation. These methods always would want to call blob endpoint and would always come in flow of blob-endpoint operation. These method will never be in dfs-endpoint operations flow.

Hence, we dont need check as such, but can add for code-sanity. Will add it.